__tests__/extension.test.ts

```diff
@@ -389,103 +389,66 @@ describe("Extension Unit Tests", () => {
-        expect(registerCommand.mock.calls[43][0]).toBe("zowe.setPrefix");
-        expect(registerCommand.mock.calls[43][1]).toBeInstanceOf(Function);
-        expect(registerCommand.mock.calls[44][0]).toBe("zowe.removeJobsSession");
-        expect(registerCommand.mock.calls[44][1]).toBeInstanceOf(Function);
-        expect(registerCommand.mock.calls[45][0]).toBe("zowe.downloadSpool");
-        expect(registerCommand.mock.calls[45][1]).toBeInstanceOf(Function);
-        expect(registerCommand.mock.calls[46][0]).toBe("zowe.getJobJcl");
-        expect(registerCommand.mock.calls[46][1]).toBeInstanceOf(Function);
-        expect(registerCommand.mock.calls[47][0]).toBe("zowe.setJobSpool");
-        expect(registerCommand.mock.calls[47][1]).toBeInstanceOf(Function);
+        expect(registerCommand.mock.calls.length).toBe(49);
+        registerCommand.mock.calls.forEach((call, i ) => {
+            expect(registerCommand.mock.calls[i][1]).toBeInstanceOf(Function);
+        });
+        const actualCommands = [];
+        registerCommand.mock.calls.forEach((call) => {
+            actualCommands.push(call[0]);
+        });
+        const expectedCommands = [
+            "zowe.addSession",
+            "zowe.addFavorite",
+            "zowe.refreshAll",
+            "zowe.refreshNode",
+            "zowe.pattern",
+            "zowe.ZoweNode.openPS",
+            "zowe.createDataset",
+            "zowe.createMember",
+            "zowe.deleteDataset",
+            "zowe.deletePDS",
+            "zowe.uploadDialog",
+            "zowe.deleteMember",
+            "zowe.removeSession",
+            "zowe.removeFavorite",
+            "zowe.safeSave",
+            "zowe.saveSearch",
+            "zowe.removeSavedSearch",
+            "zowe.submitJcl",
+            "zowe.submitMember",
+            "zowe.showDSAttributes",
+            "zowe.uss.addFavorite",
+            "zowe.uss.removeFavorite",
+            "zowe.uss.addSession",
+            "zowe.uss.refreshAll",
+            "zowe.uss.refreshUSS",
+            "zowe.uss.safeSaveUSS",
+            "zowe.uss.fullPath",
+            "zowe.uss.ZoweUSSNode.open",
+            "zowe.uss.removeSession",
+            "zowe.uss.createFile",
+            "zowe.uss.createFolder",
+            "zowe.uss.deleteNode",
+            "zowe.uss.binary",
+            "zowe.uss.text",
+            "zowe.uss.renameNode",
+            "zowe.uss.uploadDialog",
+            "zowe.zosJobsOpenspool",
+            "zowe.deleteJob",
+            "zowe.runModifyCommand",
+            "zowe.runStopCommand",
+            "zowe.refreshJobsServer",
+            "zowe.refreshAllJobs",
+            "zowe.addJobsSession",
+            "zowe.setOwner",
+            "zowe.setPrefix",
+            "zowe.removeJobsSession",
+            "zowe.downloadSpool",
+            "zowe.getJobJcl",
+            "zowe.setJobSpool"
+        ];
+        expect(actualCommands).toEqual(expectedCommands);
```

Comment:
This should make it much easier to check if all commands are there and also in expected order 😃